src/main/resources/assets/tfc/patchouli_books/book/book.json

```diff
@@ -1,6 +1,9 @@
 {
 	"name": "TerraFirmaCraft",
-	"landing_text": "A guide for TFC-TNG, a total conversion mod for 1.12$(br2)$(thing)Version 0.1$(br)Valid for TFC Version 149$(br)",
+	"landing_text": "A guide for TFC-TNG, a total conversion mod for 1.12$(br2)$(thing)Version 0.2$(br)Valid for TFC Version 149$(br)",
```

Comment:
I noticed that `name` and `landing_text` parts in `book.json` can contain localization keys. Why not use them?

Reply:
I would but I have no idea what a localization key is.

Reply:
It's like to use `"landing_text": tfc.patchouli.landing_text`, and write "tfc.patchouli.landing_text=A guide for TFC-TNG, a total conversion mod for 1.12$(br2)$(thing)Version 0.1$(br)Valid for TFC Version 149$(br)" in eh_us.lang.